ZenPacks/zenoss/ZenPackLib/migrate/FixReplacementTemplates.py

```diff
@@ -0,0 +1,62 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2023, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+
+from Products.ZenModel.migrate.Migrate import Version
+from Products.ZenModel.ZenPack import ZenPackMigration
+
+
+LOG = logging.getLogger('zen.ZenPackLib.migrate')
+
+
+class FixReplacementTemplates(ZenPackMigration):
+    """
+    Change from AristaBgpPeer plugin to AristaVrfBgpMap plugin
+    """
+
+    version = Version(1, 6, 0)
```

Comment:
The version is incorrect. Should be 2.1.3 as we are going to release the ZenPackLib 2.1.3 version.